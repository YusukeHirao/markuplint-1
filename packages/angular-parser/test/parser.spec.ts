import { parse } from 'markuplint-angular-parser'

describe('parser', () => {
  test('empty', () => {
    expect(parse('')).toMatchSnapshot()
  })
})
