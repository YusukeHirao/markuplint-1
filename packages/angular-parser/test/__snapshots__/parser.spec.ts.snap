// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parser empty 1`] = `
Object {
  "isFragment": true,
  "nodeList": Array [],
  "parseError": undefined,
}
`;
